source/main/gameplay/BeamEngine.cpp

```diff
@@ -122,51 +72,50 @@ EngineSim::~EngineSim()
     m_torque_curve = NULL;
 }
 
-void EngineSim::SetTurboOptions(int type, float tinertiaFactor, int nturbos, float param1, float param2, float param3, float param4, float param5, float param6, float param7, float param8, float param9, float param10, float param11)
+void EngineSim::SetTurboOptions(int type, float tinertiaFactor, int nturbos, float param1, float param2, float param3,
+                                float param4, float param5, float param6, float param7, float param8, float param9, float param10,
+                                float param11)
 {
-    m_engine_has_turbo = true;
+    m_engine_has_turbo  = true;
     m_engine_turbo_mode = NEW;
 
     if (nturbos > MAXTURBO)
     {
         m_num_turbos = 4;
-        LOG("Turbo: No more than 4 turbos allowed"); //TODO: move this under RigParser
+        LOG("Turbo: No more than 4 turbos allowed"); // TODO: move this under RigParser
     }
     else
         m_num_turbos = nturbos;
 
-    m_turbo_ver = type;
+    m_turbo_ver            = type;
     m_turbo_inertia_factor = tinertiaFactor;
 
-    if (param2 != 9999)
-        m_turbo_engine_rpm_operation = param2;
+    if (param2 != 9999) m_turbo_engine_rpm_operation = param2;
 
     if (m_turbo_ver == 1)
     {
         for (int i = 0; i < m_num_turbos; i++)
-            m_engine_addi_torque[i] = param1 / m_num_turbos; //Additional torque
+            m_engine_addi_torque[i] = param1 / m_num_turbos; // Additional torque
     }
     else
     {
-        float turbo_max_psi = param1; //maxPSI
-        m_max_turbo_rpm = turbo_max_psi * 10000;
+        float turbo_max_psi = param1; // maxPSI
+        m_max_turbo_rpm     = turbo_max_psi * 10000;
 
-        //Duh
+        // Duh
         if (param3 == 1)
             m_turbo_has_bov = true;
         else
             m_turbo_has_bov = false;
 
-        if (param3 != 9999)
-            m_min_bov_psi = param4;
+        if (param3 != 9999) m_min_bov_psi = param4;
 
         if (param5 == 1)
             m_turbo_has_wastegate = true;
         else
             m_turbo_has_wastegate = false;
 
-        if (param6 != 9999)
-            m_min_wastegate_psi = param6 * 10000;
+        if (param6 != 9999) m_min_wastegate_psi = param6 * 10000;

```

Comment:
I'm not sure what to do with the AllowShortIfStatementsOnASingleLine setting - I only put if statements on single line if there are multiple with similar purpose. I don't like this done to all ifs globally, and this is just ugly:
```
if (youUseClangFormat)
{ thenLongButStillShortEnoughBodiesEndUpUnindentedOnNextLine(); }
```